packages/salesforcedx-apex-replay-debugger/src/messages/i18n.ts

```diff
@@ -23,5 +23,14 @@ export const messages = {
   no_log_file_text:
     'The log file either is missing or does not have any log lines in it.',
   incorrect_log_levels_text:
-    'The log must be generated with log categories Apex code at the FINEST level and Visualforce at the FINER level.'
+    'The log must be generated with log categories Apex code at the FINEST level and Visualforce at the FINER level.',
+  unable_to_retrieve_org_info: 'Unable to retrieve OrgInfo',
+  heap_dump_error:
+    'Problems were encountered while retrieving heap dump information. Message=%s, ErrorCode=%s. Heap Dump Information=%s',
```

Comment:
Is there a reason there's a period instead of a comma after the error code message, and why there are spaces in `Heap Dump Information` but not in `ErrorCode`?

Reply:
Good catch Ruth, I'll fix them both.